wordfreq_builder/wordfreq_builder/tokenizers.py

```diff
@@ -85,7 +101,12 @@ def cld2_detect_language(text):
     #       Confidence score: float))
 
     text = CLD2_BAD_CHARS_RE.sub('', text)
-    return pycld2.detect(text)[2][0][1]
+    lang = pycld2.detect(text)[2][0][1]
+
+    # Normalize the language code: 'iw' becomes 'he', and 'zh-Hant'
+    # becomes 'zh'
+    code = langcodes.get(lang).language
+    return code
```

Comment:
Okay. Given that any change to `wordfreq_builder/tokenizers.py` necessitates like 18 hours of recomputation, I'm going to leave it as is.